FlightStreamDeck.Logics/ImageLogic.cs

```diff
@@ -231,5 +320,43 @@ public string GetGaugeImage(string text, float value, float min, float max)
 
             return "data:image/png;base64, " + base64;
         }
+
+        private void setupValue(bool top, string labelText, string value, float ratio, int img_width, float chevronSize, 
+            float width_margin, float chart_width, float max, IImageProcessingContext ctx
+        )
+        {
+            if (labelText.Length > 0)
```

Comment:
```
if (labelText.Length == 0) 
{
    return;
} 
```

Would save you a bunch of indents :)

Reply:
Yes I would prefer "return" style on top for parameter validation. Also, for string, string.IsNullOrWhiteSpace would normally catch all scenarios that you want to skip.

Reply:
I'm of the camp a function should only have one return statement, period.

If y'all want me to short circuit stuff, I'll try to overcome that habit.